views/form.js

```diff
@@ -5,17 +5,33 @@ let database = firebase.database();
 let storage = firebase.storage();
 
 export function createForm (data) {
-	let imageNumber = 1;
 	createPage1 (data);
-	let diaper = {}
+	let diaper = {};
+	diaper.images =[];
 	$('#input-image').change(function(event) {
 		if (event.target.files.length > 0) {
 			showPreview (event);
-			addImageToStorage (imageNumber, diaper);
-			imageNumber = imageNumber + 1;
+			addProfileImageToStorage (diaper);
+		}
+	});
+	let image = {'image-number': 1}
+	addImageInputTemplate (image);
+	let currentImageInput = 'input-image-' + 1;
+	$('#' + currentImageInput).change(function(event) {
+		console.log('frytki')
```

Comment:
I assume console.logs will be removed eventually? ;)
Is there still some problem persisting that you're trying to debug with these console.logs?